Cargo.toml

```diff
@@ -239,6 +240,11 @@ component = [
   'dep:wasmparser',
   'dep:serde_json',
 ]
+simd = [
+  'wasmparser/simd',
+  'wasmprinter/simd',
+  'wasm-encoder/simd',
+]
```

Comment:
I think for this, sort of like wasm-encoder below, let's go ahead and just unconditionally enable simd for these dependencies at the CLI level perhaps?